pyopenetl/operations.py

```diff
@@ -344,6 +375,10 @@ def seed_table(self, read_table: str, read_chunksize: int, write_table: str) ->
             gc.collect()
 
             its += 1
+        index_creation_query = self.get_indices_from_heroku(read_table=read_table, write_table=write_table)
+        with self.dest_conn.connect() as cloud_sql_conn:
+            # in order to do upserts later, each table needs to have a unique constraint on the primary key
```

Comment:
I'd remove this comment -- seems like a duplicate of another earlier comment and doesn't really apply here